src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,68 @@
+package seedu.addressbook.ui;
+
+/**
+ * Formatter class to format text based on data structures
```

Comment:
Glad to see you added header comment for the class =).
Though, what do you mean by `based on data structures?`.
Is there any other way to improve the comment?